client/lib/feed-stream-store/index.js

```diff
@@ -16,30 +16,15 @@ import wpcom from 'lib/wp';
 
 const wpcomUndoc = wpcom.undocumented();
 
-function feedKeyMaker( post ) {
-	return {
-		feedId: post.feed_ID,
-		postId: post.ID
-	};
-}
-
-function siteKeyMaker( post ) {
-	return {
-		blogId: post.site_ID,
-		postId: post.ID
-	};
-}
+// by throwing out blogId if feed_item_ID exist then we know if its a feed/blog from here on out
+const keyMaker = post => ( {
```

Comment:
also, we already know if it's a feed or a blog. if it's a blog, it has a blog id. otherwise, it's a feed.